Cargo.toml

```diff
@@ -54,7 +54,8 @@ path = "ad9959"
 
 [dependencies.stm32h7xx-hal]
 features = ["stm32h743v", "rt", "unproven", "ethernet", "quadspi"]
-git = "https://github.com/stm32-rs/stm32h7xx-hal"
+git = "https://github.com/quartiq/stm32h7xx-hal"
+branch = "feature/stabilizer-dma"
```

Comment:
We should document why this is using a branch. My current reasons are:
* incorporates `dma` branch from official repo
* Removes the `Drop` trait from DMA streams

We can likely use the `dma` branch from the source repository directly as well, which may be easier.